packages/cactus-plugin-persistence-fabric/package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "@hyperledger/cactus-plugin-persistence-fabric",
+  "version": "1.1.3",
```

Comment:
@barnapa You'll need to update the version to match the current latest (v1.1.4 as I'm writing this).

The same goes for all the cross-package dependencies declarations here in both `dependecies` and `devDependencies`.